src/main/steps/edge-case/dashboard/template.njk

```diff
@@ -29,6 +29,9 @@
   <li>
     <a class="govuk-link" href="/c100-rebuild">C100 Application</a>
   </li>
+  <li>
+    <a class="govuk-link" href="/c100-rebuild/get-case">C100 GetCase</a>
```

Comment:
We are not using this edge-case/dashboard any more as pvt law team switched to new dashboard inside /prl-cases.
For temp we are using prl-cases/dashboard-v1 for c100 rebuild

Reply:
Made this change